Source/Shaders/ModelExperimental/ModelSilhouetteStageFS.glsl

```diff
@@ -0,0 +1,5 @@
+void silhouetteStage(out vec4 color) {
```

Comment:
We did some debugging and determined that this should be `inout`. Otherwise, it's undefined behavior as to what the value of `color` is if not updated. And it seems like the drivers in Linux do something different than in Windows :shrug: 

```suggestion
void silhouetteStage(inout vec4 color) {
```